config/streams/dublin.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "pandemic51.core.streaming.M3U8Stream",
+    "webpage": "https://www.earthcam.com/world/ireland/dublin/?cam=templebar",
+    "chunk_path": "https://d3o4twxzdiwvsf.cloudfront.net/fecnetwork/4054.flv/chunklist.m3u8",
+    "GMT": "0"
+}
```

Comment:
Is it ok for some of these streams to be missing `stream_name`

Reply:
yes, it's automatically populated if it's missing